ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer/Kernel.hs

```diff
@@ -114,6 +121,11 @@ passiveWalletLayerCustomDBComponent logFunction keystore acidDB = do
             , pwlGetDBSnapshot         = liftIO $ Kernel.getWalletSnapshot wallet
             , pwlLookupKeystore        = \hdrId -> liftIO $
                 Keystore.lookup (WalletIdHdSeq hdrId) (wallet ^. Kernel.walletKeystore)
+            , pwlRemoveKey             = \rootIds -> liftIO $
+                Kernel.removeKeyFromKeyStore wallet rootIds
+            , pwlGetUnknownKeys        = liftIO $ getUnknownKeys wallet
```

Comment:
Those functions are defined for abstract `MonadIO`, so using `liftIO` here is redundant.
Define them as `IO` or remove `liftIO` here.